### PR TITLE
Added fundamentals

### PR DESCRIPTION
**Pull request by remarcmij**
- loops
- scope

### CODE REVIEW

File: Week2/MAKEME.md [hunk 1]
```diff
@@ -3,29 +3,29 @@
 ```
 Topics discussed in class this week:
 • Intro JavaScript (What is it, where can you use it for)
-• Variables [var, let, const]
-• Basic Data types [Strings, Numbers, Arrays, Booleans]
+• Variables (var, let, const)
+• Basic Data types (strings, numbers, arrays, booleans)
```
Comment (mkruijt) [inline Week2/MAKEME.md]: hi @remarcmij why lowercase?

File: README.md [hunk 2]
```diff
@@ -4,24 +4,24 @@
...
-|7.|• Object Oriented Programming  <br>• Code flow (order of execution) <br>• Async VS Sync|[Reading Week 7](/Week7/README.md)|[Homework Week 7](/Week7/MAKEME.md)|[Review](/Week7/REVIEW.md)|
-|8.|• Structure for a basic SPA <br>• XHTTP Requests <br>• API calls|[Reading Week 8](/Week8/README.md)|[Homework Week 8](/Week8/MAKEME.md)|Review|
-|9.|• (re)writing data structures (in JSON) <br>• Promises|[Reading Week 9](/Week9/README.md)|[Homework Week 9](/Week9/MAKEME.md)|[Review](/Week9/REVIEW.md)|
+|Week|Topic|Read|Homework|
+|----|-----|----|--------|
+|0.|Preparation for your first JavaScript session|[Pre-reading](/Week0/README.md)|-|
+|1.|Git Session with Unmesh|[Reading Week 1](/Week1/README.md) | [Homework Week 1](/Week1/MAKEME.md)|
+|2.|• Intro JavaScript (What is it, where can you use it for)<br>• [Variables (var, let, const)](fundamentals/variables.md)<br>• [Basic Data types (strings, numbers, arrays, booleans)](fundamentals/values.md)<br>• [Operators](fundamentals/operators.md)<br>• [Naming conventions](fundamentals/naming_conventions.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
```
Comment (mkruijt) [inline README.md]: Why did you change (strings, numbers, arrays, booleans) to lowercase?

File: README.md [hunk 3]
```diff
@@ -4,24 +4,24 @@
...
+|3.|• Git work flow :smiling_imp:<br>• [Advanced data types (objects)](fundamentals/objects.md) <br>• [Conditional execution](fundamentals/conditional_execution.md) <br>• [Statements vs Expressions](fundamentals/statements_expressions.md)<br> • [Loops (for/while)](fundamentals/loops.md)<br>• [Functions](fundamentals/functions.md) <br>• [Scope](fundamentals/scope.md)|[Reading Week 3](/Week3/README.md)|[Homework Week 3](/Week3/MAKEME.md)|
```
Comment (mkruijt) [inline README.md]: same question as above

File: Week2/MAKEME.md [hunk 4]
```diff
@@ -115,6 +117,9 @@ console.log('I'm awesome');
   let foo = 3;
   console.log('The value of my variable foo is: ' + foo);
   ```
+
+  (Curious to know what a `foo` is? Check [this article](https://en.wikipedia.org/wiki/Metasyntactic_variable) on WikiPedia.)  
+
```
Comment (mkruijt) [inline Week2/MAKEME.md:122]: nice addition!

**Review by mkruijt — CHANGES_REQUESTED** [review]
Hi @remarcmij thank you for rearranging the reviews. Can you explain why you changed the some of the topics to lowercase?

File: README.md (hunk 2 above)
Comment (remarcmij) [reply README.md]: @mkruijt String, Number and Boolean refer to the constructor functions. The lowercase versions refer to the primitive values. But if you prefer to stick to uppercase, that's fine with me too.

```
> typeof Number
'function'
> typeof 10
'number'
>
```

**Review by mkruijt — APPROVED** [review]
